src/api/schemas/repos.js

```diff
@@ -48,8 +48,7 @@ export const repoSchema = new schema.Entity(
   'repos',
   {},
   {
-    idAttribute: repo =>
-      (repo.full_name ? repo.full_name : repo.name).toLowerCase(),
+    idAttribute: repo => (repo.full_name ? repo.full_name : repo.name),
```

Comment:
I'm assuming there's no harm to just keeping the repo name as is (without lowercase) as the idAttribute right? (definitely better this way but just want to double check)

Reply:
I remember some edge cases where the case was a problem, but definitely can't tell what it was now. 
Let's drop the lowercase and see what happens :)

Reply:
@housseindjirdeh  @machour Thanks for your concerns. @machour was inspired from [redux examples](https://github.com/reduxjs/redux/blob/master/examples/real-world/src/middleware/api.js#L52). It reads user input as keyword to fetch Github data. And when finding entities, it also converts the key to lower cases, see [link](https://github.com/reduxjs/redux/blob/master/examples/real-world/src/containers/UserPage.js#L76). Because Github APIs accept case-insensitive arguments and return case-sensitive data, as I commented in https://github.com/gitpoint/git-point/issues/826#issuecomment-434563624.

I think our use cases are a bit different. We always use case-sensitive ids. So we should stop converting entity ids to lower cases.

Reply:
That clears things up nicely, thank you makes perfect sense :)